carbonmark/components/pages/Users/SellerConnected/index.tsx

```diff
@@ -41,7 +41,8 @@ export const SellerConnected: FC<Props> = (props) => {
     props.userAddress,
     {
       network,
-      expiresAfter: address === props.userAddress ? "0" : undefined,
+      expiresAfter:
+        address?.toLowerCase() === props.userAddress ? "0" : undefined,
```

Comment:
for safety's sake please convert both to lowercase explicitly

```suggestion
      expiresAfter:
        address?.toLowerCase() === props.userAddress.toLowerCase() ? "0" : undefined,
```